docker/requirements.txt

```diff
@@ -55,4 +55,3 @@ wheel>=0.31.0
 # yarl 1.6.0 broke query string parsing in aiohttp 3.6.[012] https://github.com/aio-libs/aiohttp/issues/4972#issuecomment-700290809
 yarl<1.6.0
 zulip==0.6.3
-six>1.14.0
```

Comment:
this is no longer needed because the new pip install feature will always upgrade to compatible versions if necessary and fail if incompatible versions exist.